src/3d/chunks/qgschunkedentity_p.cpp

```diff
@@ -689,6 +686,11 @@ QgsChunkQueueJob *QgsChunkedEntity::startJob( QgsChunkNode *node )
 {
   if ( node->state() == QgsChunkNode::QueuedForLoad )
   {
+    if ( !mIsValid )
+    {
+      node->cancelQueuedForLoad();
+      return nullptr;
+    }
```

Comment:
I am not sure if I understand why it is necessary to add this - my understanding is that when jobs get stopped, no new jobs should get started before the entity gets deleted...